cluster/images/kubelet/Dockerfile

```diff
@@ -1,8 +1,13 @@
-FROM centos
-ADD kubelet /kubelet
+FROM debian:jessie
+
+COPY kubelet /kubelet
 RUN chmod a+rx /kubelet
-RUN cp /usr/bin/nsenter /nsenter
 
+COPY kubelet-runner.sh /kubelet-runner.sh
+RUN chmod a+rx /kubelet-runner.sh
+
```

Comment:
I would say that it's best to install the same packages as `hyperkube` has. https://github.com/kubernetes/kubernetes/blob/master/cluster/images/hyperkube/Dockerfile#L3
Otherwise, the risk is very high that some package (that we already solved in `/hyperkube kubelet`) is required to have, and another PR must fix that.

We should include the packages in this PR